lldb/scripts/framework-header-include-fix.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+# Usage: framework-header-include-fix.py <path/to/input-header.h> <path/to/output-header.h>
```

Comment:
Add a short description of what this script does. Mainly, what is it fixing specifically?